bin/PythonApps/EPLaunch.app/Contents/MacOS/EPLaunch

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+SCRIPT_DIR="$( cd -- "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 ; pwd -P )"
+echo "Script Path: $SCRIPT_DIR"
+EP_ROOT_DIR=$(dirname "$(dirname "$(dirname "$SCRIPT_DIR")")")
+if [[ "$DEBUG" = 1 ]]; then
+  result=`osascript -e "display dialog \"Script Path: $SCRIPT_DIR\nEP_ROOT_DIR: $EP_ROOT_DIR\" buttons {\"Ok\"}"`
+fi;
```

Comment:
If you define DEBUG=1 you'll get a debug dialog.

<img width="416" alt="image" src="https://github.com/user-attachments/assets/52c09edc-20fa-465a-b4c7-f687a94dc22d" />